src/ImageGallery.jsx

```diff
@@ -582,11 +582,15 @@ export default class ImageGallery extends React.Component {
 
             onTouchStart={event => this.slideToIndex.call(this, index, event)}
             onClick={event => this.slideToIndex.call(this, index, event)}>
-
-            <img
-              src={item.thumbnail}
-              alt={item.thumbnailAlt}
-              onError={onThumbnailError.bind(this)}/>
+            <div style={{ width: '63px' }}>
+              <img
+                src={item.thumbnail}
+                alt={item.thumbnailAlt}
+                onError={onThumbnailError.bind(this)}/>
+               <span style={{ fontSize: '0.8em', color: 'black' }}>
```

Comment:
I tried adding styles with className in scss xiaolin, But it didn't work. The build was successful and i could see my style classNames in build/image_gallery.css The stylings i added were not displayed in the browser.
code snippet:

src/image-gallery.scss
`.image-gallery-div-thumbnail {
  width: 63px;
}

.image-gallery-label {
  font-size: 0.8em;
  color: $ig-black;
}
`
src/ImageGallery.jsx
`          <a
            onMouseOver={this._handleMouseOverThumbnails.bind(this, index)}
            onMouseLeave={this._handleMouseLeaveThumbnails.bind(this, index)}
            key={index}
            className={
              'image-gallery-thumbnail' +
              (currentIndex === index ? ' active' : '') +
              thumbnailClass
            }

            onTouchStart={event => this.slideToIndex.call(this, index, event)}
            onClick={event => this.slideToIndex.call(this, index, event)}>
            <div className="image-gallery-div-thumbnail">
            <img
              src={item.thumbnail}
              alt={item.thumbnailAlt}
              onError={onThumbnailError.bind(this)} />
            <span className="image-gallery-label">
              {item.label}
            </span>
            </div>
          </a>
`